### PR TITLE
Iteration2

### PR DESCRIPTION
**Pull request by kabcouwer**
What I did:
-Added to seeds file
-Added Brewery create feature
-Added Brewery update feature
-Added Beer create feature
-Added Brewery update feature
- Beer index page only shows 'true' records
- Added link to sort the Brewery's beers list alphabetically
- Cleaned up tests
- Added what user story each test is associated with for better project organization

Testing: 
- all tests passing

What to do:
- nothing. iteration 2 is complete!

### CODE REVIEW

File: app/views/beers/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%= form_with url: "/beers/#{@beer.id}", method: :patch, local: true do |form| %>
+<%= form.label :name, 'Name:' %>
```
Comment (JasonPKnoll) [inline app/views/beers/edit.html.erb:2]: Fun fact. You can have these pre filled out with the previous information by adding ```, value: "#{@beer.name}"```. Not necessary but a fun addition.

File: spec/features/beers/show_spec.rb [hunk 2]
```diff
@@ -38,33 +42,65 @@
...
+  end
+
+  it 'links to the update page, updates the beer, and redirects to the beer show page' do
+    #user story 14
+    brewery = Brewery.create!(name: "Goldspot Brewing Co",
+                              location: "Denver, CO",
+                              year_established: 2016,
+                              multiple_brewhouses: false
+                            )
+    beer = brewery.beers.create!(name: "Gay IPA",
```
Comment (JasonPKnoll) [inline spec/features/beers/show_spec.rb:80]: Please tell me this is real. I would definitely order this for simply existing..

**Review by JasonPKnoll — APPROVED** [review]
Your tests are really thorough, and your code looks clean! Great work!I left one specific comment about a potential functionality that can be added onto the edit.html's but they are not required for the project. Just fun additions.

**Comment by kabcouwer** [comment]
approved so just moving into main :)

**Pull request merged**